connectors/debezium-mysql-source/3.1.0.5/debezium-mysql-source.md

```diff
@@ -0,0 +1,156 @@
+---
+description: The Debezium source connector pulls messages from MySQL and persists the messages to Pulsar topics.
+author: ["ASF"]
+contributors: ["ASF"]
+language: Java
+document: 
+source: "https://github.com/streamnative/pulsar/tree/v3.1.0.5/pulsar-io/debezium/mysql"
+license: Apache License 2.0
+tags: ["Pulsar IO", "Debezium", "MySQL", "Source"]
+alias: Debezium MySQL Source
+features: ["Use Debezium MySQL source connector to sync data to Pulsar"]
+license_link: "https://www.apache.org/licenses/LICENSE-2.0"
+icon: "/images/connectors/debezium.jpg"
+download: "https://github.com/streamnative/pulsar/releases/download/v3.1.0.5/pulsar-io-debezium-mysql-3.1.0.5.nar"
+support: StreamNative
+support_link: https://streamnative.io
+support_img: "/images/streamnative.png"
+owner_name: ""
+owner_img: ""
+dockerfile:
+sn_available: true
+id: "debezium-MySQL-source"
+---
+
+The MySQL source connector pulls messages from MySQL and persists the messages to Pulsar topics by using debezium.
+
+![](/images/connectors/mysql-source.png)
+
+## Quick start
+
+### Prerequisites
+
+The prerequisites for connecting a Debezium MySQL source connector to external systems include:
+
+1. Create a MySQL service: This connector uses the debezium v1.9, Please refer to this [document](https://debezium.io/releases/1.9/) to see the compatible MySQL versions.
+2. Prepare MySQL Database: Please refer to this [document](https://debezium.io/documentation/reference/1.9/connectors/mysql.html#setting-up-mysql) to complete the prepare steps on MySQL.
+
+{% callout title="Note" type="note" %}
+If you are using AWS MySQL service, you need to use the [params group](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithParamGroups.html) to set the [binlog_format](https://debezium.io/documentation/reference/1.9/connectors/mysql.html#enable-mysql-binlog) to `ROW`.
+{% /callout %}
+
+
+### 1. Create a table on MySQL
+Run the following SQL command on your MySQL. If you don't require the `before` data, you can disregard the configuration of `REPLICA IDENTITY`.
```

Comment:
```suggestion
Run the following SQL command on your MySQL.
```

We may need to delete this statement because it's PostgreSQL-only config. https://github.com/streamnative/pulsar-hub/pull/325/files#r1388786231